src/commands/apply/index.ts

```diff
@@ -18,6 +19,16 @@ import {
   trackApplyCommandFailed,
   trackApplyCommandStarted,
 } from '../../analytics'
+import { ProfilingIntegration } from '@sentry/profiling-node'
+import { config } from '../../config'
+
+Sentry.init({
+  dsn: 'https://5bc27276ac684a56bab07632be10a455@o2239.ingest.sentry.io/4505312653410304',
+  tracesSampleRate: 1.0,
+  profilesSampleRate: 1.0,
+  integrations: [new ProfilingIntegration()],
+  environment: config.environment,
+})
```

Comment:
Should we move the config out somewhere and share it between apply and [create](https://github.com/contentful/contentful-merge/blob/main/src/commands/create/index.ts#L27-L33)?